src/org/jitsi/impl/neomedia/rtcp/RTCPReceiverFeedbackTermination.java

```diff
@@ -410,7 +410,9 @@ private RawPacket doTransform(RawPacket pkt, boolean send)
             {
                 ByteArrayBuffer baf = it.next();
                 int pt = RTCPHeaderUtils.getPacketType(baf);
-                if (pt == RTCPRRPacket.RR || RTCPREMBPacket.isREMBPacket(baf))
+                if (pt == RTCPRRPacket.RR
+                        || RTCPREMBPacket.isREMBPacket(baf)
+                        || RTCPTCCPacket.isTCCPacket(baf))
```

Comment:
What do you think about adding a "continue" here, or changing the second "if" to an "else if"? Calling remove() twice is illegal (at least for this iterator), and the code is unclear that it doesn't happen.

Reply:
You're right, I just patched the code.